src/main/java/de/tum/in/www1/artemis/web/rest/dto/metrics/ExerciseInformationDTO.java

```diff
@@ -17,14 +17,15 @@
  * @param type      the type of the exercise
  */
 @JsonInclude(JsonInclude.Include.NON_EMPTY)
-public record ExerciseInformationDTO(long id, String shortName, String title, ZonedDateTime start, ZonedDateTime due, Class<? extends Exercise> type) {
+public record ExerciseInformationDTO(long id, String shortName, String title, ZonedDateTime start, ZonedDateTime due, Double maxPoints, Class<? extends Exercise> type) {
```

Comment:
If I get the endpoint now on my PR I get:
```
{
  "type" : "https://www.jhipster.tech/problem/problem-with-message",
  "title" : "Internal Server Error",
  "status" : 500,
  "detail" : "Cannot instantiate class 'de.tum.in.www1.artemis.web.rest.dto.metrics.ExerciseInformationDTO' (it has no constructor with signature [java.lang.Long, java.lang.String, java.lang.String, java.time.ZonedDateTime, java.time.ZonedDateTime, java.lang.Class], and not every argument has an alias)",
  "path" : "/api/metrics/course/1/student",
  "message" : "error.http.500"
}
```

Do you know why that is?

Reply:
I think the issue is with `findAllExerciseInformationByCourseId`.

Also, is the `maxPoints` compatible with the `score` and `averageScore`? I probably have to divide by 100 and multiply with the `maxPoints` if I understand correctly? 
Would this also handle bonus points?